lms/djangoapps/instructor_task/tests/test_integration.py

```diff
@@ -148,31 +176,31 @@ def test_rescoring_option_problem(self):
         self.submit_student_answer('u3', problem_url_name, [OPTION_2, OPTION_1])
         self.submit_student_answer('u4', problem_url_name, [OPTION_2, OPTION_2])
 
-        self.check_state('u1', descriptor, 2, 2, 1)
-        self.check_state('u2', descriptor, 1, 2, 1)
-        self.check_state('u3', descriptor, 1, 2, 1)
-        self.check_state('u4', descriptor, 0, 2, 1)
+        self.check_state(self.user1, descriptor, 2, 2)
+        self.check_state(self.user2, descriptor, 1, 2)
+        self.check_state(self.user3, descriptor, 1, 2)
+        self.check_state(self.user4, descriptor, 0, 2)
 
-        # update the data in the problem definition
-        self.redefine_option_problem(problem_url_name)
-        # confirm that simply rendering the problem again does not result in a change
-        # in the grade:
+        # update the data in the problem definition so the answer changes.
+        self.redefine_option_problem(problem_url_name, **problem_edit)
+
+        # confirm that simply rendering the problem again does not change the grade
         self.render_problem('u1', problem_url_name)
-        self.check_state('u1', descriptor, 2, 2, 1)
+        self.check_state(self.user1, descriptor, 2, 2)
 
         # rescore the problem for only one student -- only that student's grade should change:
         self.submit_rescore_one_student_answer('instructor', problem_url_name, User.objects.get(username='u1'))
-        self.check_state('u1', descriptor, 0, 2, 1)
-        self.check_state('u2', descriptor, 1, 2, 1)
-        self.check_state('u3', descriptor, 1, 2, 1)
-        self.check_state('u4', descriptor, 0, 2, 1)
+        self.check_state(self.user1, descriptor, new_expected_score, new_expected_max, 1)
+        self.check_state(self.user2, descriptor, 1, 2)
+        self.check_state(self.user3, descriptor, 1, 2)
+        self.check_state(self.user4, descriptor, 0, 2)
 
         # rescore the problem for all students
         self.submit_rescore_all_student_answers('instructor', problem_url_name)
-        self.check_state('u1', descriptor, 0, 2, 1)
-        self.check_state('u2', descriptor, 1, 2, 1)
-        self.check_state('u3', descriptor, 1, 2, 1)
-        self.check_state('u4', descriptor, 2, 2, 1)
+        self.check_state(self.user1, descriptor, new_expected_score, new_expected_max)
+        self.check_state(self.user2, descriptor, 1, new_expected_max)
+        self.check_state(self.user3, descriptor, 1, new_expected_max)
+        self.check_state(self.user4, descriptor, new_expected_score_4, new_expected_max)
```

Comment:
Here's the commit of the individual change if you'd like to take a look - as I'm going to squash now.